scss/_base_code.scss

```diff
@@ -1,6 +1,8 @@
 @import 'settings';
 
 $color-pre-bg: rgba($color-x-dark, 0.03);
+$digit-width: 0.5rem; // measured width of one character in the monospaced font
```

Comment:
Could we use `1ch` here?

Reply:
TIL about `ch`... however, unfortunately not, in the next line we're combining `$digit-width` with a `rem` value, `$sph-inner`, and `ch` and `rem` values are incompatible.

Reply:
that's what calc is for e.g. `calc(2rem + 4cm)` :)

Reply:
Ah yes of course! Cool, adding this in now.